package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "tsc",
     "prepublishOnly": "npm run build",
-    "test": "mocha --parallel",
+    "test": "globstar -- node --loader tsx --test \"test/*.spec.ts\"",
```

Comment:
```
node --loader tsx --test test/
```

I think it might work like this no?

Reply:
Maybe just put the two files to execute (always better than rely on another package)

```
node --loader tsx --test test/badge.spec.ts, test/result.spec.ts
```

Something like that